src/include/simeng/memory/MMU.hh

```diff
@@ -19,7 +19,7 @@ namespace memory {
 
 class MMU {
  public:
-  MMU(const uint16_t latency, VAddrTranslator fn);
+  MMU(VAddrTranslator fn);
 
```

Comment:
I cant comment on the function itself, but given `tick()` now does nothing it can be removed